### PR TITLE
Update color.md

### PR DESCRIPTION
**Pull request by claycrenshaw**
Style edits.

### CODE REVIEW

File: src/content/guidelines/accessibility/color.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
-**WCAG guidelines contain recommendations for making the web accessible to users with disabilities, including color blindness or low vision.**
+**IBM is committed to serving all users, regardless of visual ability.**
 
 ## Introduction
-Web and software experiences should be accessible for everyone, regardless of a person's abilities or impairments. Visual impairments can range in severity, from low vision, to color blindness, to complete blindness. The Carbon team has done extensive research and testing to ensure each component was designed with accessibility as our top priority.
+Visual impairments can inculde low vision, color blindness, and complete blindness. Carbon components have been designed to accommodate the entire spectrum of visual impairment, though designer diligence is still needed to ensure that the components are used correctly.
```
Comment (joshblack) [inline src/content/guidelines/accessibility/color.md]: inculde -> include

File: src/content/guidelines/accessibility/color.md [hunk 2]
```diff
@@ -27,11 +26,11 @@ Color blindness involves difficulty in perceiving or distinguishing between colo
 | Monochromacy | All colors |  
 
 ### Designing for color blindness
-When designing color palettes, it's always good to run them through a color blind simulator. If you're working in Sketch, we recommend the [Stark](http://www.getstark.co/) plugin.
+When designing with color, it's always good to run your pages or comps through a color-blind simulator to ensure that all functions are still clear and usable by the color-blind user. If you're working in Sketch, we recommend the [Stark](http://www.getstark.co/) plugin.
```
Comment (joshblack) [inline src/content/guidelines/accessibility/color.md]: comps feels casual in this context, would it help to consolidate this phrase to just `good to run your work`?

File: src/content/guidelines/accessibility/color.md [hunk 3]
```diff
@@ -40,18 +39,18 @@ Low vision can include partial sight in one or both eyes, and range from mild to
 | Type | Color deficiency |
 |------| ----------------|
 | Low acuity | Also known as **blurred vision.** Can make text difficult to read, since it would appear fuzzy. |
-| Low contrast sensitivity | The ability to determine fine detail and distinguish one object from another.     |
+| Low-contrast sensitivity | The ability to determine fine detail and distinguish one object from another.     |
```
Comment (joshblack) [inline src/content/guidelines/accessibility/color.md]: Just curious, what's the rule for the hyphen for some of these terms?

File: src/content/guidelines/accessibility/color.md [hunk 4]
```diff
@@ -40,18 +39,18 @@ Low vision can include partial sight in one or both eyes, and range from mild to
...
 _[Source: NoCoffee - Visual impairments ](https://accessgarage.wordpress.com/2013/02/09/458/)_
 
 ![blurred vision and tunnel vision on Google.com](images/color-accessibility-2.png)
 
-_Low vision disabilities_
+_Low-vision disabilities_
 
 ### Designing for low vision
-To get an understanding of the various low vision disabilities, we recommend using the [NoCoffee](https://chrome.google.com/webstore/detail/nocoffee/jjeeggmbnhckmgdhmgdckeigabjfbddl) Chrome plugin to preview websites. Low vision users may be using a screen reader to preview your website or experience, so ensure you're following [keyboard guidelines](/guidelines/accessibility/keyboard) to ensure the page is read to the user in a logical order.
+To get an understanding of the various low-vision disabilities, we recommend using the [NoCoffee](https://chrome.google.com/webstore/detail/nocoffee/jjeeggmbnhckmgdhmgdckeigabjfbddl) Chrome plugin to preview websites. Low-vision users may be using a screen reader to preview your website or experience, so ensure you're following [keyboard guidelines](/guidelines/accessibility/keyboard) to ensure the page is read to the user in a logical order.
```
Comment (joshblack) [inline src/content/guidelines/accessibility/color.md]: Would be fun to plug http://www.getstark.co/ here, as well.

Comment (claycrenshaw) [reply src/content/guidelines/accessibility/color.md]: It's in there.

**Pull request merged**